lib/plugins/sfDoctrinePlugin/lib/record/sfDoctrineRecord.class.php

```diff
@@ -207,17 +207,22 @@ public function __call($method, $arguments)
   }
 
   /**
-   * Get the Doctrine date value as a PHP DateTime object
+   * Get the Doctrine date value as a PHP DateTime object, null if the value is not set
    *
    * @param string $dateFieldName   The field name to get the DateTime object for
-   * @return DateTime $dateTime     The instance of PHPs DateTime
+   * @return DateTime|null $dateTime     The instance of PHPs DateTime
```

Comment:
Can you add an empty line before this ```@return``` tag? ([Like this](https://github.com/marekk/symfony1/blob/85b74b025b47af4af8bf7a304fdedb418016ebe1/lib/plugins/sfDoctrinePlugin/lib/record/sfDoctrineRecord.class.php#L135))